### PR TITLE
fix(ui): adapt tabs styles

### LINKED ISSUE #48: Fix tabs style
**Problem**
Design differs for different instances (task modal, candidate page, position page)
On low resolutions text is centered for selectize (maybe separate issue)

**Solution**
Like in Tabler, but no background on active tab

### PR DESCRIPTION
**Pull request by wrmk**
Closes #48.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/layouts/ats/profile.html.slim [hunk 1]
```diff
@@ -6,7 +6,7 @@
         = yield :header
 
       - if content_for?(:tab_list)
-        section#tab-list.row.mt-2
```
Comment (wrmk) [inline app/views/layouts/ats/profile.html.slim:9]: fixed this problem, line moved left
image

File: app/views/ats/tasks/_show.html.slim [hunk 2]
```diff
@@ -42,7 +42,7 @@ ruby:
                       }
               .row.mt-3
                 .col-12
-                  ul.nav.nav-tabs rolw="tablist"
```
Comment (wrmk) [inline app/views/ats/tasks/_show.html.slim:45]: fixed typo

File: app/views/ats/positions/_profile.html.slim [hunk 3]
```diff
@@ -10,45 +10,54 @@
   = render "ats/positions/side_header"
 / TAB LIST
 - content_for :tab_list
+  ruby:
```
Comment (wrmk) [inline app/views/ats/positions/_profile.html.slim:13]: adapted like in candidate's profile
also solved problem with tasks counter that did not displayed in mobile version

File: app/views/ats/candidates/_profile.html.slim [hunk 4]
```diff
@@ -77,9 +77,10 @@
         = link_to tab_labels[:activities], tab_ats_candidate_path(@candidate, :activities),
                   class: "nav-link #{'active' if @active_tab == 'activities'}"
   .d-md-none.dropdown.w-100.profile-mobile-nav
-    button.btn.form-control.d-flex.align-items-center.text-dark.dropdown-toggle.w-100.px-3[
+    button.btn.form-control.dropdown-toggle.justify-content-between.w-100[
         id="dropdownNavLink" data-bs-toggle="dropdown"]
-      = tab_labels[@active_tab.to_sym]
+      span
```
Comment (wrmk) [inline app/views/ats/candidates/_profile.html.slim:82]: necessary to keep counter (tasks, files etc) near tab name

File: app/views/ats/candidates/_profile.html.slim [hunk 5]
```diff
@@ -77,9 +77,10 @@
         = link_to tab_labels[:activities], tab_ats_candidate_path(@candidate, :activities),
                   class: "nav-link #{'active' if @active_tab == 'activities'}"
   .d-md-none.dropdown.w-100.profile-mobile-nav
-    button.btn.form-control.d-flex.align-items-center.text-dark.dropdown-toggle.w-100.px-3[
```
Comment (wrmk) [inline app/views/ats/candidates/_profile.html.slim:80]: removed  styles that do not affect styles

**Review by ksnitsky — APPROVED** [review]

**Pull request merged**